docs/api.md

```diff
@@ -4552,4 +4816,50 @@ These are in alphabetic order, with the exception that the `Extended` versions o
 ## Property Detailed (object)
 + name: `first_name` (string, required) - The name of the Property.
 + publishedAt: `2018-01-21T00:04:11.153Z` (string, required) - Publishing timestamp of the form that defined this property for the first time.
-+ forms: (array[Form KeyValue]) - List of forms that create the property
\ No newline at end of file
++ forms: (array[Form KeyValue]) - List of forms that create the property
+
+## EntitySummaryFields (object)
++ uuid: `uuid:85cb9aff-005e-4edd-9739-dc9c1a829c44` (string, required) - The `uuid` of the Entity that uniquely identifies the Entity.
++ datasetName: `people` (string, required) - Name of the dataset
++ createdAt: `2018-01-19T23:58:03.395Z` (string, required) - ISO date format. The time that the server received the Entity.
++ updatedAt: `2018-03-21T12:45:02.312Z` (string, optional) - Timestamp of the last update in ISO date format. `null` when there is only one version of the Entity.
++ deletedAt: `2018-03-21T12:45:02.312Z` (string, optional) - Timestamp of the deletion in ISO date format. `null` if the Entity is not deleted.
```

Comment:
Could we include `creatorId` here as well? Right now, the individual entity endpoint will return the creator of the `currentVersion`, but it'd be useful if it could also return the original creator of the entity itself.

Reply:
In addition to `creatorId`, could it also return `creator` when extended metadata is requested?

Reply:
agree